dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -445,7 +445,7 @@ def stddev(self) -> float:
         """Return stddev value."""
         if self.match_count == 0:
             return np.nan
-        return np.sqrt(self.variance)
+        return cast(float, np.sqrt(self.variance))
```

Comment:
I think we convert later to a float bc this could be an np.float64. Probably good to say this could be np.float so that we are returning the real type.

Reply:
I see. This would still require a `cast` though because numpy's type annotations for `sqrt` on a scalar has a return type of `Any`.

Reply:
I'm trying to cast it to `np.float64` but the issue becomes that there's not really a compatible return type annotation because `np.nan` is of type `float`. Using something like `np.float_` causes mypy to complain about the `np.nan` case and sticking with `float` causes mypy to complain about the `np.sqrt` case.

Reply:
Changing it to a Union causes issues with line 385:
```
            "stddev": utils.find_diff_of_numbers(self.stddev, other_profile.stddev),
```
```
dataprofiler/profilers/numerical_column_stats.py:385: error: Value of type variable "T" of "find_diff_of_numbers" cannot be "object"
```

Reply:
So `def stddev(self) -> Union[float, np.float64]:`
with the `cast(np.float64, np.sqrt(self.variance))` doesn't work?

Reply:
Yes. It seems to cause issues with other functions.

Reply:
I would assume `def stddev(self) -> Union[float, np.float64]:` even w/o the cast should be functional.

but if this is erroring elsewhere, then we should fix that once we do this correctly.